src/backend/setup.cfg

```diff
@@ -26,6 +26,7 @@ classifiers =
 install_requires =
     arrow==1.2.1
     Brotli==1.0.9
+    boto3==1.20.26
```

Comment:
You install `boto3` because the swift API is S3 compatible ?

Reply:
Yes, Django connects to the Swift backend as if it was an S3 bucket. 
boto3 is necessary for django storages to work.